src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+@import "../utils/variables";
+
+.nav__list {
+  display: flex;
+  height: 100%;
+  list-style: none;
+}
+
+.nav__item {
+  height: 100%;
+
+  &:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  height: 100%;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: $main-accent;
+  text-decoration: none;
+}
+
+.is-active {
+  color: $blue-accent;
+  position: relative;
+}
+
+.nav__link--hover:hover {
+  color: $blue-accent;
+}
+
```

Comment:
Add a transition property to the '.nav__link' class to make changes smooth on hover as mentioned in the task description.